### ISSUE TITLE
Ability to change "MT" icon when seeing original touchbar buttons

### ISSUE THREAD

**Issue by matoelorriaga** [Toxblh/MTMR#140]
It would be nice to be able to change the "MT" icon :)

**Comment by mjabdulm** [comment]
I agree, it would be nicer to show a clock instead of a static "MT" icon when seeing original Touch Bar buttons.

**Comment by Toxblh** [comment]
The nice idea!

**Comment by ChefAustin** [comment]
+1 to this.

Additionally, a toggleable option to hide the tray icon would be great!

**Comment by mjabdulm** [comment]
Hi @Toxblh any possibility of this idea?

**Comment by mjabdulm** [comment]
> I agree, it would be nicer to show a clock instead of a static "MT" icon when seeing original Touch Bar buttons.
Just a bump to check the feasibility of having a clock instead of MT icon on the Touch Bar.
